noxfiles/dev_nox.py

```diff
@@ -176,7 +176,7 @@ def fides_env(session: Session, fides_image: Literal["test", "dev"] = "test") ->
 
     session.log("Installing ethyca-fides locally...")
     install_requirements(session)
-    session.install("-e", ".")
+    session.install("-e", ".", "--no-deps")
```

Comment:
How will this work... aren't the deps needed to run `fides` CLI? We can't assume they'll be there in a fresh `nox` venv?

Reply:
Note the line above it, this is the standardized way to install the requirements in nox sessions

Reply:
Aha, OK!

Reply:
"i sped up the install process by removing the line that installs things" wouldn't be the first time an engineer has done it 🙃